docs/api/structures/web-preferences.md

```diff
@@ -146,6 +146,11 @@
 * `hideScrollBars` boolean (optional) - Sets the Chromium WebPreferences
    hide_scrollbars field. Default is `false`.
 * `enablePinchZoom` boolean (optional) - Sets whether to enable pinch to zoom in Chromium. Default is `true`.
+* `windowTransform` string (optional) - Specifies whether to apply a transform to the window. Default is `none`. Accepted values are
```

Comment:
You can ignore this if you want to, but it would be good to mention  _Linux_ here.

Reply:
Done